recipes/openassetio/all/conanfile.py

```diff
@@ -0,0 +1,177 @@
+            minimum_version = self._compilers_minimum_version.get(str(self.info.settings.compiler), False)
+            if minimum_version and Version(self.info.settings.compiler.version) < minimum_version:
+                raise ConanInvalidConfiguration(
+                    f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
+                )
+
+    def build_requirements(self):
+        self.tool_requires("cmake/3.21.7")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+
+        tc.variables["OPENASSETIO_ENABLE_TESTS"] = not self.conf.get("tools.build:skip_test", default=True, check_type=bool)
+
+        libcxx = self.settings.get_safe("compiler.libcxx")
+        if libcxx is not None:
+            if libcxx == "libstdc++11":
+                tc.variables["OPENASSETIO_GLIBCXX_USE_CXX11_ABI"] = True
+            else:
+                tc.variables["OPENASSETIO_GLIBCXX_USE_CXX11_ABI"] = False
+
+        if self.options.without_python:
+            tc.variables["OPENASSETIO_ENABLE_PYTHON"] = False
+        else:
+            tc.variables["OPENASSETIO_ENABLE_PYTHON"] = True
+            tc.variables["Python_EXECUTABLE"] = self._python_exe
+            if is_msvc(self):
+                tc.variables["Python_LIBRARY"] = self._python_windows_lib
+
+        tc.generate()
+        tc = CMakeDeps(self)
+        tc.generate()
+        tc = VirtualBuildEnv(self)
+        tc.generate(scope="build")
+
+    @property
+    def _python_exe(self):
+        return pathlib.Path(self.deps_user_info["cpython"].python).as_posix()
+
+    @property
+    def _python_windows_lib(self):
+        pth = pathlib.Path(
+            self.deps_cpp_info["cpython"].rootpath,
+            self.dependencies["cpython"].cpp_info.components["embed"].libdirs[0],
+            self.dependencies["cpython"].cpp_info.components["embed"].libs[0])
+        pth = pth.with_suffix(".lib")
+        return pth.as_posix()
+
+    def build(self):
+        apply_conandata_patches(self)
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package_id(self):
+        if not self.options.without_python:
+            self.info.requires["cpython"].minor_mode()
+
+    def package(self):
+        copy(self, pattern="LICENSE", dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
+        cmake = CMake(self)
+        cmake.install()
+
+        rm(self, "OpenAssetIOConfig*.cmake", os.path.join(self.package_folder, "lib", "cmake", "OpenAssetIO"))
+        rm(self, "OpenAssetIOTargets*.cmake", os.path.join(self.package_folder, "lib", "cmake", "OpenAssetIO"))
+        rm(self, "*.pdb", os.path.join(self.package_folder, "lib"))
+        rm(self, "*.pdb", os.path.join(self.package_folder, "bin"))
+
+    def package_info(self):
+        self.cpp_info.libs = ["package_lib"]
```

Comment:
```suggestion
        self.cpp_info.libs = ["package_lib"]
```
Is it correct?

Reply:
Looks like a copy-paste error, good spot, thanks.

Reply:
Removed the erroneous entry in d2256b7e